packages/webcert/src/store/api/apiMiddleware.test.ts

```diff
@@ -150,4 +150,23 @@ describe('Test API middleware', () => {
       expect(throwErrorAction?.payload.type).toEqual(ErrorType.SILENT)
     })
   })
+
+  describe('handle functionBlocker', () => {
```

Comment:
Would be good to include tests that verifies that the functionalBlockerType is called as "a pair" no matter if the api call is a success or it causes an error.

The implementation in the apiMiddleware looks solid, but it is good if the tests verifies that no one breaks this important logic.